ci-operator/config/jrangelramos/func/jrangelramos-func-presumtest.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+#images:
+#  - dockerfile_path: openshift/ci-operator/gitserver-image/Dockerfile
+#    to: gitserver-image
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: GOFLAGS="" make test
+  container:
+    from: src
+- as: e2e-oncluster-test
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
```

Comment:
Please use `serverless-ci` like we do in https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__415.yaml#L121-L127
We're migrating away from openshift-ci cluster pool.

Also the `releases` section should match version of this cluster pool. Like here: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__415.yaml#L107-L111